lib/MusicBrainz/Server/Edit/Release/ReorderMediums.pm

```diff
@@ -1,4 +1,5 @@
 package MusicBrainz::Server::Edit::Release::ReorderMediums;
+use utf8;
```

Comment:
It only needs adding when you use UTF-8 chars in the source code, and I used some fancy quotes in the exception message ("Can’t")